pkg/httphelper/httphelper.go

```diff
@@ -0,0 +1,57 @@
+package httphelper
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+// shamelessly taken from martini-contrib/render/render.go
+const (
+	ContentType = "Content-Type"
+	ContentJSON = "application/json"
+)
+
+type ResponseHelper interface {
+	Error(error)
+	JSON(int, interface{})
+	WriteHeader(int)
+}
```

Comment:
The pkg is based on that, I copied the code and removed the martini render.Render requirement and made generic, I could try merging the 2 together to make one generic package.